### PR TITLE
Fabo/csp

### PR DESCRIPTION
**Pull request by faboweb**
Breaking:
This will disable the functionality to set the backend via query parameters in production!

Test:
- build for production: `yarn build:ui`
- run dist files: `http-server ./app/dist`
- build for mainnet: `STARGATE=https://rpc.lunie.io RPC=https://rpc.lunie.io CIRCLECI=true yarn build:ui`
- run dist files: `http-server ./app/dist`

**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: webpack.renderer.config.js [hunk 1]
```diff
@@ -7,7 +7,9 @@ const webpack = require(`webpack`)
 const fs = require(`fs`)
 
 const HtmlWebpackPlugin = require(`html-webpack-plugin`)
+const CSPWebpackPlugin = require(`csp-webpack-plugin`)
 const VueLoaderPlugin = require(`vue-loader/lib/plugin`)
+const MiniCssExtractPlugin = require(`mini-css-extract-plugin`)
 // const BundleAnalyzerPlugin = require(`webpack-bundle-analyzer`)
```
Comment (jbibla) [inline webpack.renderer.config.js]: delete comments?

File: webpack.renderer.config.js [hunk 2]
```diff
@@ -126,11 +133,39 @@ const rendererConfig = {
...
+      'style-src': [`'self'`, `'unsafe-inline'`],
+      'connect-src':
+        !production ? `*` : [
+          // third party tools
+          `https://sentry.io`,
+          `https://appzi-collector-b.azurewebsites.net`,
+          `https://keybase.io`,
+          // mainnet
+          `https://stargate.cosmos.network`,
+          `wss://rpc.cosmos.network:26657`,
```
Comment (jbibla) [inline webpack.renderer.config.js]: https://rpc.lunie.io

File: webpack.renderer.config.js [hunk 3]
```diff
@@ -126,11 +133,39 @@ const rendererConfig = {
...
+      // 'style-src': production ? `'self'` : `*`, // SECURITY Appzi is applying styles inline, inquired to them already
+      'style-src': [`'self'`, `'unsafe-inline'`],
+      'connect-src':
+        !production ? `*` : [
+          // third party tools
+          `https://sentry.io`,
+          `https://appzi-collector-b.azurewebsites.net`,
+          `https://keybase.io`,
+          // mainnet
+          `https://stargate.cosmos.network`,
```
Comment (jbibla) [inline webpack.renderer.config.js]: https://stargate.lunie.io

File: webpack.renderer.config.js [hunk 4]
```diff
@@ -126,11 +133,39 @@ const rendererConfig = {
...
+          `wss://rpc.cosmos.network:26657`,
+          // testnet
+          `https://sntajlxzsg.execute-api.eu-central-1.amazonaws.com/`,
```
Comment (jbibla) [inline webpack.renderer.config.js]: would be great to have a dns entry for our testnet node (and we should move this over too).

File: webpack.renderer.config.js [hunk 5]
```diff
@@ -126,11 +133,39 @@ const rendererConfig = {
...
+          `wss://test.voyager.ninja:26657`
```
Comment (jbibla) [inline webpack.renderer.config.js]: is this even still working? should be moved over to lunie infra.

File: webpack.renderer.config.js (hunk 1 above)
Comment (faboweb) [reply webpack.renderer.config.js]: can be uncommented to actually analyze the bundle size, which we probably want to do from time to time

**Pull request merged**